pkg/addons/addons.go

```diff
@@ -54,6 +54,10 @@ var Force bool = false
 // Currently only used for gcp-auth
 var Refresh bool = false
 
+// AutoPauseTime is used to specify how often minikube needs to check to autpause
+// cluster.
+var AutoPauseTime time.Duration = time.Minute * 1
```

Comment:
AutoPauseTime is not persisted anywhere. This means whenever the minikube binary starts, this will be reset to 1 minute. You need to store this in the Kubernetes config.